lms/resources/_js_config/file_picker_config.py

```diff
@@ -63,6 +65,14 @@ def canvas_config(cls, request, application_instance):
             },
         }
 
+        if pages_enabled:
+            config["listPages"] = {
```

Comment:
This won't be present if listPages is false. Not sure if the type over config.ts reflects this correctly.

Reply:
Does this mean the property would not be present at all, or it would be but with value `null`?

In any case, I see this is defined as mandatory in the FE type, while it should be either `listPages?: APICallInfo` if it can be optionally skipped, or `listPages: APICallInfo | null` if the value can be null.